projects/kit/components/radio/radio.style.less

```diff
@@ -31,6 +31,7 @@
     position: absolute;
     background-color: currentColor;
     border-radius: 100%;
+    animation: scaleIn 300ms;
```

Comment:
`.transition(transform)` will take care of duration

Reply:
Fixed!